src/Microsoft.Extensions.Logging.EventSource/project.json

```diff
@@ -0,0 +1,49 @@
+{
+  "version": "1.0.0-*",
+  "description": "A logger provider that sends data logged through ILogger interfaces to EventSource/EventListener infrastructure",
+  "packOptions": {
+    "tags": [
+      "EventSource",
+      "logging",
+      "ETW"
+    ],
+    "repository": {
+      "type": "git",
+      "url": "git://github.com/aspnet/logging"
+    }
+  },
+  "buildOptions": {
+      "warningsAsErrors": true,
+      "keyFile": "../../tools/Key.snk",
+      "xmlDoc": true
+  },
+  "dependencies": {
+    "Microsoft.Extensions.Logging.Abstractions": "1.0.0-*",
+    "System.Diagnostics.Tracing": "4.1.0-*",
+    "System.Threading": "4.0.11-*",
+    "Newtonsoft.Json": "8.0.3"
+  },
+  "frameworks": {
+    "net46": {
+      "dependencies": {
+        "System.Runtime": "4.0.0"
+      },
+      "imports": [ "dotnet" ]
+    },
+    "netcoreapp1.0": {
```

Comment:
You still can't use `netcoreapp`, it's the wrong TFM. Use `imports` instead: https://github.com/aspnet/Mvc/blob/3d494fd7326d3e137fdb22b53f01c5a34866fbaa/src/Microsoft.AspNetCore.Mvc.ViewFeatures/project.json#L59-L61

(Mind you, this is all temporary; once a new build of Json.NET is out that supports `netstandard`, this will go away.)